contracts/BSCValidatorSet.sol

```diff
@@ -120,7 +111,7 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     }
 
     /*----------------- cross chain package -----------------*/
-    struct IbcValidatorSetPackage {
+    struct ValidatorSetPackage {
         uint8 packageType;
         Validator[] validatorSet;
         bytes[] voteAddrs;
```

Comment:
event like `validatorJailed` are deprecated, please mark them and check other events.